### PR TITLE
(wip) build demo app

### PR DESCRIPTION
**Pull request by jasonLaster**
This app could replace the code section as the demo of how Marionette works.

The idea is that you see one row of contibutors and then click a big button to see the code that powers it.

### CODE REVIEW

File: src/js/demo.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+window.onload = function() {
+
+  var Application = Marionette.Application.extend({
```
Comment (jasonLaster) [inline src/js/demo.js]: So... this would be the code we'd show when they want to see what a marionette app looks like. It better be good :)

**Comment by samccone** [comment]
this is really cool @jasonLaster I think we can do some neat UX with like a toggle button to show code, but great idea :clap:

**Comment by jasonLaster** [comment]
Yea. Excited to see what we come up with. 
@jdaudier right up your alley

**Comment by jdaudier** [comment]
Adding how other apps show code as reference (feel free to add more):

* https://www.mapbox.com/mapbox.js/example/v1.0.0
* http://semantic-ui.com/elements/button.html
* http://vuejs.org/examples/grid-component.html
* http://canjs.com *(I kinda like being able to see the code right next to the app)*
* http://famo.us/university/lessons/#/famous-101/displaying/1

**Comment by jdaudier** [comment]
Closing this since it's over a year ago, and it seems like no one has the time to finish this currently.

**Pull request closed without merging**